app/js/domains.json

```diff
@@ -28,6 +28,7 @@
   {"name": "ethereumpurse", "version": "0.9", "registrar": "0xfa6512358d241b636edf5f8ccaaf86676dec21a7"},
   {"name": "ishername", "version": "0.9", "registrar": "0xfa6512358d241b636edf5f8ccaaf86676dec21a7"},
   {"name": "satoshiwallet", "version": "0.9", "registrar": "0xfa6512358d241b636edf5f8ccaaf86676dec21a7"},
+  {"name": "sendmecrypto", "version": "0.9", "registrar": "0xfa6512358d241b636edf5f8ccaaf86676dec21a7"},
```

Comment:
Unless you signed this over to the ensnow version of the registrar, this line should read like the first couple - eg, version 1.0, no address.

Reply:
I just finished signing it over following [this] (https://medium.com/@weka/how-to-list-your-domain-on-ensnow-7297808f31f5) guide.  Re-reading the guide it looks like it probably should be like you suggested.